### PR TITLE
705-bug-libegl-warning-egl-failed-to-create-dri2-screen

### PR DESCRIPTION
**Pull request by abasb123**
This pull request addresses the issue of the "Failed to Create DRI2 Screen" error, which occurs when running the Deakin Detonator Toolkit (DDT) on virtual machines, particularly when using Kali Linux on a Windows 10 host via VirtualBox.

### CODE REVIEW

**Comment by michaeljpigott** [comment]
Hi @abasb123, you need to merge main into your branch, so you have the Documentation folder. You can then save your libgl-bug-fix into that folder. 

I think the steps you need to follow are: 
1. `git pull` (Make sure your main branch is up-to-date). 
2. `git checkout 705-bug-libegl-warning-egl-failed-to-create-dri2-screen-1`
3. `git rebase main`

I find [this stackoverflow chat](https://stackoverflow.com/questions/41045548/merging-changes-from-master-into-my-branch) quite useful. 

Kind regards

Michael

**Comment by michaeljpigott** [comment]
Placed this pull request on hold while waiting for main to be rebased into the branch.

**Comment by abasb123** [comment]
> Hi @abasb123, you need to merge main into your branch, so you have the Documentation folder. You can then save your libgl-bug-fix into that folder.
> 
> I think the steps you need to follow are:
> 
> 1. `git pull` (Make sure your main branch is up-to-date).
> 2. `git checkout 705-bug-libegl-warning-egl-failed-to-create-dri2-screen-1`
> 3. `git rebase main`
> 
> I find [this stackoverflow chat](https://stackoverflow.com/questions/41045548/merging-changes-from-master-into-my-branch) quite useful.
> 
> Kind regards
> 
> Michael

Hi @michaeljpigott , I have completed the steps. Kindly review my pull request. 
Regards, 
Abbas.

**Comment by michaeljpigott** [comment]
Added @JacksonGIThuba as reviewer.

File: libgl-bug-fix.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+#BUG-FIX-LIBGL-ERROR.
+#Review the " failed to create dri2" error message
+
+# Identify the envionment(Kali Linux on vm with windows 10)
+
+#Each command should be run on the command line one by one for best results
+
```
Comment (JacksonGIThuba) [inline libgl-bug-fix.md:7]:
```suggestion When running for me the first and only error occurred as a result of yarn not being installed
#ensure that you are in the DDT directroy
yarn install
#after installing enter the following 
yarn run tauri dev
```

**Review by JacksonGIThuba — APPROVED** [review]
Easy to follow tutorial for those who have the issue, will be a massive time saver for other who come across the same issue

**Pull request merged**